sunpy/data/_sample.py

```diff
@@ -8,8 +8,8 @@
 from sunpy.util.parfive_helpers import Downloader
 
 _base_urls = (
-    'http://data.sunpy.org/sunpy/v1/',
     'https://github.com/sunpy/sample-data/raw/master/sunpy/v1/',
+    'http://data.sunpy.org/sunpy/v1/',
```

Comment:
Do we want to swap this?

Reply:
Questions, Questions. I think given we basically have unlimited github LFS hosting now, I am going to say yes, but we should probably re-review the situation before the 2.1 release.